commands/login.go

```diff
@@ -15,7 +15,11 @@ var (
 	LoginCmd = &cobra.Command{
 		Use:   "login",
 		Short: "Authenticate your nanobox client with your nanobox.io account.",
-		Long:  ``,
+		Long:  `
+		Authenticate with your nanobox account by passing the username and password in or using the following environment variables:
```

Comment:
I suggest removing the leading spaces on these 3 lines, so the output isn't pushed out so much.